internal/services/mssql/mssql_server_security_alert_policy_resource.go

```diff
@@ -91,11 +93,9 @@ func resourceMsSqlServerSecurityAlertPolicy() *pluginsdk.Resource {
 			"state": {
 				Type:     pluginsdk.TypeString,
 				Required: true,
-				ValidateFunc: validation.StringInSlice([]string{
-					string(sql.SecurityAlertPolicyStateDisabled),
-					string(sql.SecurityAlertPolicyStateEnabled),
-					string(sql.SecurityAlertPolicyStateNew),
-				}, false),
+				// NOTE: 'SecurityAlertsPolicyStateNew' no longer exists in the new API...
```

Comment:
We can remove this comment as it's not relevant unless looking at this PR

Reply:
Fixed.